terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -4,6 +4,7 @@ aws_profile = "staging-eu-west-2"
 account_local_url = "https://staging.company-information.service.gov.uk"
 feature_flag_wle = "false"
 feature_flag_ap01_web = "true"
+feature_flag_ch01_web = "true"
```

Comment:
Can we set this to be false in the first instance please. We don't want any CH01 features being in there whilst we are doing pen testing etc.